store/store_action.go

```diff
@@ -40,7 +40,7 @@ type RefreshOptions struct {
 	// RefreshManaged indicates to the store that the refresh is
 	// managed via snapd-control.
 	RefreshManaged bool
-	IsAutoRefresh  bool
+	Scheduled      bool
```

Comment:
any particular reason to do this? maybe we want this but as its own PR, here it just makes the overall change noisier

Reply:
I changed it because it's now set when it's an auto-refresh but not if it's a continuation of one. I can leave it as it was or split it out, do you have a preference?